db/migrate/20220811085951_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] On edr diagram, user is called as `author` on other tables:

<img width="1013" alt="Screenshot 2022-09-14 at 21 42 11" src="https://user-images.githubusercontent.com/77038610/190236886-8fa7ed54-0c8d-40ea-b418-24f75095e14d.png">

You can achieve that by changing the migration to: 
FROM:
`t.references :user, null: false, foreign_key: true`
TO:
`t.references :author, foreign_key: { to_table: 'users' }`

Reply:
Implemented 👍